modules/statistics/server/controllers/statistics.server.controller.js

```diff
@@ -12,9 +12,105 @@ var path = require('path'),
     User = mongoose.model('User');
 
 /**
+ * Get count of all public users
+ */
+exports.getUsersCount = function(callback) {
+  User.count({ public: true }, function(err, count) {
+    callback(err, count || 0);
```

Comment:
I'm not 100% sure about this, but I think the general pattern is to invoke callbacks with either `callback(err)` or `callback(null, value)`. This (and line 65 below) will return a count of `0` if there is an error. That seems unusual to me, although I'll gladly defer to anyone who has a stronger opinion on this.